guides/source/sign_up.md

```diff
@@ -0,0 +1,1652 @@
+**DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON <https://guides.rubyonrails.org>.**
+
+Sign Up and Registration
+========================
+
+This guide covers adding Sign Up and Registration to the store e-commerce application in the [Getting Started Guide](getting_started.html)). We will use the final code from that guide as a starting place.
+
+After reading this guide, you will know:
+* How to add Sign Up
+* How to rate limit controller actions
```

Comment:
Is this how you say it, or would it be better to say something like:
Apply rate limits to controller actions

Reply:
```suggestion
* Rate limit controller actions
```